app/src/main/java/org/coepi/android/ui/onboarding/OnboardingCard.kt

```diff
@@ -0,0 +1,17 @@
+package org.coepi.android.ui.onboarding
+
+
+sealed class OnboardingCard(val title: String, val message: CharSequence) {
+
+    class SmallOnboardingCard(
+        title: String,
+        message: CharSequence,
+        val order: Int
```

Comment:
I'd put `order` in all the cards. This seems to make assumptions about where the cards are based on style, which isn't good.